meta-balena-honister/recipes-containers/tini/tini_0.14.0.bbappend

```diff
@@ -0,0 +1,2 @@
+# Thud static PIE seems broken as binary gets into segfault at runtime
```

Comment:
Let's review this, probably not required for Honister. If it is, let's rename thud to honister to make it clear it applies to honister.

Reply:
Thank for pointing this out Alex, I removed it in https://github.com/balena-os/meta-balena/pull/2368/commits/76b383d394537872f8f3968a13451f5e8e7228a6 and haven't spotted any segfault with 2 local builds, one for TX2 and another for Xavier AGX. Multi container test app worked, I could access each container's webterminal and saw no failed services. I'm not sure how it would've manifested when tini segfaulted but I assume the engine would've complained/failed, if it were to boot to that point..

```
root@75cd96c:~# systemctl | grep fail
root@75cd96c:~# cat /etc/os-release && uname -r
ID="balena-os"
NAME="balenaOS"
VERSION="2.85.2+rev4"
VERSION_ID="2.85.2+rev4"
PRETTY_NAME="balenaOS 2.85.2+rev4"
MACHINE="jetson-tx2"
META_BALENA_VERSION="2.86.0"
BALENA_BOARD_REV="a24cd70"
META_BALENA_REV="67911cca"
SLUG="jetson-tx2"
4.9.253-l4t-r32.6
```

Reply:
thanks @acostach seems safe to remove it